### DISCUSSION TITLE
Locked organizations ?

### DISCUSSION THREAD

**Discussion by aryanshridhar** [github/india#639] [Students]
Hey, Whenever I visit the organizations panel, I see that most of them are locked. By locked, I mean that the `Learn more` button is disabled unlike a few others which are unlocked.
May I know what does this mean ? Is it that these organizations aren't yet registered for the externship ?

Thanks in advance :)

**Comment by KSSSenapati** [comment]
I think they have **closed** receiving applications for their projects, that's why they might have disabled the '**Learn more**' button

**Comment by punitjain03** [comment]
We have disabled the button because, We are adding the projects and detailing for the same. Keep an eye soon it will be unlocked for all

**Reply by PranavRishi** [reply]
Hello sir, I want to know one thing please. 
I m a beginner in tech world. 
So will I be taught when I apply for them, Or do I need to have a very good knowledge!?
